pkg/scheduler/factory.go

```diff
@@ -214,11 +211,43 @@ func (c *Configurator) create() (*Scheduler, error) {
 		NextPod:         internalqueue.MakeNextPodFunc(podQueue),
 		Error:           MakeDefaultErrorFunc(c.client, podQueue, c.schedulerCache),
 		StopEverything:  c.StopEverything,
-		VolumeBinder:    c.volumeBinder,
 		SchedulingQueue: podQueue,
 	}, nil
 }
 
+func appendIfNotExist(pcs []schedulerapi.PluginConfig, config schedulerapi.PluginConfig) []schedulerapi.PluginConfig {
+	for _, pc := range pcs {
+		if pc.Name == config.Name {
+			return pcs
+		}
+	}
+	return append(pcs, config)
+}
+
+func inPlugins(plugins *schedulerapi.Plugins, name string) bool {
```

Comment:
I didn't find any function I can use to check if a plugin is configured in `schedulerapi.Plugins`, so I add this temporary function to make `TestPluginsConfigurationCompatibility/disable_some_default_plugins` happy.

Reply:
If we inject the plugin config no matter if it is enabled, all wanted plugin configs in tests must include `volumebinding` plugin config. That may look weird. I prefer injecting plugin config only if the plugin is enabled in plugin sets. Anyway, this function will be removed soon when we eliminate `ComponentConfig.BindTimeoutSeconds`.

Reply:
That's a good point. Let's leave it like this for now.